gui/src/main/java/org/jboss/wise/gwt/client/view/EndpointConfigView.java

```diff
@@ -87,7 +87,6 @@
    public EndpointConfigView() {
 
       SimplePanel contentDetailsDecorator = new SimplePanel();
-      contentDetailsDecorator.setWidth("100%");
```

Comment:
Just as a confirmation, is this on purpose? Asking as the change does not look related to WISE-289